test/cans/prod/86e7b58e-b9f0-4020-8b34-c61d6da02d44/2019-09-20T103932.395795Z/metadata.json

```diff
@@ -220,6 +220,7 @@
             "document_id": "2e64e936-85ec-49a0-8365-578b5621bcb1",
             "submission_date": "2019-09-20T08:29:52.277Z",
             "update_date": "2019-09-20T08:38:58.167Z",
+            "submitter_id": "67a720af-4482-4619-81d7-3693b2d3cc4c",
```

Comment:
Does the referenced schema version support this field?

Reply:
Yes, these supplementary files have the schema https://schema.humancellatlas.org/type/file/2.2.0/supplementary_file which contains a `provenance` described by https://schema.humancellatlas.org/system/1.1.0/provenance in which `submitter_id` is an optional field.